packages/expo/bundledNativeModules.json

```
@@ -0,0 +1,72 @@
+{
+  "@expo/vector-icons": "^9.0.0",
+  "@react-native-community/netinfo": "1.3.0",
+  "@unimodules/core": "~1.0.0",
+  "@unimodules/react-native-adapter": "~1.0.0",
+  "expo-ads-admob": "~4.0.0",
+  "expo-ads-facebook": "~4.0.0",
+  "expo-analytics-amplitude": "~4.0.0",
+  "expo-analytics-segment": "~4.0.0",
+  "expo-app-auth": "~4.0.0",
+  "expo-app-loader-provider": "~4.0.0",
+  "expo-av": "~4.0.0",
+  "expo-background-fetch": "~4.0.0",
+  "expo-barcode-scanner": "~4.0.0",
+  "expo-blur": "~4.0.0",
+  "expo-brightness": "~4.0.0",
+  "expo-calendar": "~4.0.0",
+  "expo-camera": "~4.0.0",
+  "expo-constants": "~4.0.0",
+  "expo-contacts": "~4.0.0",
+  "expo-crypto": "~4.0.0",
+  "expo-document-picker": "~4.0.0",
+  "expo-face-detector": "~4.0.0",
+  "expo-facebook": "~4.0.0",
+  "expo-file-system": "~4.0.0",
+  "expo-font": "~4.0.0",
+  "expo-gl": "~4.0.0",
+  "expo-gl-cpp": "~4.0.0",
+  "expo-google-sign-in": "~4.0.0",
+  "expo-haptics": "~4.0.0",
+  "expo-image-manipulator": "~4.0.0",
+  "expo-image-picker": "~4.0.0",
+  "expo-intent-launcher": "~4.0.0",
+  "expo-keep-awake": "~4.0.0",
+  "expo-linear-gradient": "~4.0.0",
+  "expo-local-authentication": "~4.0.0",
+  "expo-localization": "~4.0.0",
+  "expo-location": "~4.0.0",
+  "expo-mail-composer": "~4.0.0",
+  "expo-media-library": "~4.0.0",
+  "expo-payments-stripe": "~4.0.0",
+  "expo-permissions": "~4.0.0",
+  "expo-print": "~4.0.0",
+  "expo-random": "~4.0.0",
+  "expo-secure-store": "~4.0.0",
+  "expo-sensors": "~4.0.0",
+  "expo-sharing": "~2.0.0",
+  "expo-sms": "~4.0.0",
+  "expo-speech": "~4.0.0",
+  "expo-sqlite": "~4.0.0",
+  "expo-task-manager": "~4.0.0",
+  "expo-web-browser": "~4.0.0",
+  "lottie-react-native": "2.5.0",
+  "react-native-branch": "2.2.5",
+  "react-native-gesture-handler": "^1.1.0",
```

```suggestion
  "react-native-gesture-handler": "~1.1.0",
```
I know it was just copied from `expo`'s `package.json` (we should fix it there), but our native deps shouldn't be tied with `^`. In terms of unimodules and kmagiera's libs, it's safe to use `~`.